components/Header.vue

```diff
@@ -5,7 +5,7 @@ const { logout } = useAuthStore();
 const { isLoggedIn } = storeToRefs(useAuthStore());
 
 const handleLogout = async () => {
-  logout();
+  await logout();
```

Comment:
why await here?

Reply:
weird I don't even remember adding it :/ Removing it now.